addons/mail/models/mail_channel.py

```diff
@@ -107,6 +107,7 @@ def default_get(self, fields):
     channel_partner_ids = fields.Many2many('res.partner', 'mail_channel_partner', 'channel_id', 'partner_id', string='Listeners')
     channel_message_ids = fields.Many2many('mail.message', 'mail_message_mail_channel_rel')
     is_member = fields.Boolean('Is a member', compute='_compute_is_member')
+    is_channel_admin = fields.Boolean('Is the admin', compute='_compute_is_channel_admin')
```

Comment:
If the "Confirm message" is not *super*-necessary, I would remove this field to keep this model as simple as possible